app/src/main/java/com/androidimpact/app/Ingredient.java

```diff
@@ -20,6 +20,7 @@
  * @version 1.0
  */
 public class Ingredient implements Serializable  {
+    protected String id;
```

Comment:
For an ID attribute, can you add a `@DocumentId` tag on top? It helps Firebase know how to serialize ingredients when reading/writing from firestore.

I think something like 

```
public class Ingredient implements Serializable  {
    @DocumentID
    protected String id;
    protected String description;
    protected float amount;
```

Reply:
Ok sike, easy fix and not a big deal so I just changed it